### DISCUSSION TITLE
base-devel question

### DISCUSSION THREAD

**Discussion by neolib** [msys2/MINGW-packages#8791] [General]
Many times I got confused with packages in MSYS2 and MINGW32/64. Now I have base-devel installed in MSYS2. But I also found that mingw-w64-i686-* also provides similar files such as make/gdb/etc. Should I uninstall base-devel in MSYS2 and install mingw-w64-i686-* ones in MINGW32/64?

**Comment by Biswa96** [comment]
tl;dr no. The tools are separated by `$PATH` variable.
